README.md

```diff
@@ -128,6 +135,8 @@ Also, add the framework in `Build Phases -> Embed Frameworks` and on `Link Binar
 
 ![Adding WebRTC local reference in Pods target](assets/WebRTC-References-Pods.png)
 
+###### For Simulator only (*):
+
 - In `Frameworks Search Paths`, insert the following line: 
 ```
 $(PROJECT_DIR)/../libWebRTC/WebRTC.xcframework/tvos-arm64_x86_64-simulator
```

Comment:
Here it should be `tvos-x64-simulator` in order to include both simulator archs. Same in the `Header Search Paths`replacing `tvos-arm64` with `tvos-x64-simulator`